src/main/kotlin/no/nav/rekrutteringsbistand/stillingssokproxy/Main.kt

```diff
@@ -8,6 +11,30 @@ val Any.log: Logger
 
 fun log(name: String): Logger = LoggerFactory.getLogger(name)
 
+val environment = dotenv { ignoreIfMissing = true }
+
 fun main() {
     log("main").info("Starter applikasjon")
+
+    val port = environment["JAVALIN_PORT"].toInt()
+    val aliveUrl = "http://localhost:${port}/internal/isAlive"
+    val readyUrl = "http://localhost:${port}/internal/isReady"
+
+    val javalin = Javalin.create().start(port)
+
+    if (environment["NAIS_CLUSTER_NAME"] == "local") {
+        log("main").warn("Applikasjonen settes opp med konfigurasjon for lokal kjøring")
+        javalin.before { context ->
+            context.cookieStore("innloggetVeileder", Security.InnloggetVeileder("brukernavn", "visningsnavn", "navident"))
+        }
+    } else {
+        val security = Security()
+        security.lagSikkerhetsfilter(javalin, listOf(aliveUrl, readyUrl))
+    }
```

Comment:
Kanskje vi burde dele opp lokal kjøring og kjøring i miljø i to main-funksjoner?
Hvor alt som kun kjøres lokalt ligger i test-sources, så kan vi sende avhengigheter inn i en felles funksjon ala`App.start(security: Security, <andre avhengigheter>)`.

Har gjort det slik i indekseren og synes det har fungert veldig bra!

Reply:
Vi har nå en egen fil for lokal kjøring. Den har en main-metode. Eneste forskjell fra "vanlig" kjøring er at den ikke setter opp sikkerhetsfilter. Kan du ta en titt på det? Mulig Joar ønsker å gjøre endringer der..